Design.md

```diff
@@ -3,6 +3,26 @@
 This document captures the design decisions made after discussion in issues. When a design issue is closed, the
 conclusion should be summarized here with a link to the issue.
 
+## OSTree Delivery Format
+
+- Originally discussed in issue [#23](https://github.com/coreos/fedora-coreos-tracker/issues/23). 
+
+### Summary
+
+RPM-OStree supports three delivery models for delivering content to
+end user systems.
+
+- rojig: uses a special rojig RPM and re-assembles OSTree commit from RPMs
+  already on mirrors.
+- OCI: OSTree commits are packaged up in OCI container images and delivered 
+  via a container registry
+- OSTree Repo: OSTree commits are stored in an OSTree repo (like a git
```

Comment:
Minor: I'd put this first, since it's the "standard" option most people are probably familiar with.

Reply:
Super minor: maybe `OSTree remote` instead?

Everything goes in an OSTree repo in the end :)

Reply:
I can move it to be first

Reply:
I think I'll leave it as OSTree repo if you don't mind :). I added some text to the description to explain the OSTree repo is server side.